docs/modules/ROOT/pages/servlet/appendix/namespace/http.adoc

```diff
@@ -2168,6 +2168,9 @@ Allows injection of the ExpiredSessionStrategy instance used by the ConcurrentSe
 Maps to the `maximumSessions` property of `ConcurrentSessionControlAuthenticationStrategy`.
 Specify `-1` as the value to support unlimited sessions.
 
+[[nsa-concurrency-control-session-limit-strategy-ref]]
+* **session-limit**
```

Comment:
Please call this `max-sessions-ref` so as to clarify that it is a strategy bean for determining the session limit.